.github/workflows/tests.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
             GOARCH=amd64 PASSES='build' ./test.sh
             GOARCH=386 PASSES='build' ./test.sh
             GO_BUILD_FLAGS='-v -mod=readonly' GOOS=darwin GOARCH=amd64 ./build.sh
+            GO_BUILD_FLAGS='-v -mod=readonly' GOOS=darwin GOARCH=arm64 ./build.sh
             GO_BUILD_FLAGS='-v -mod=readonly' GOOS=windows GOARCH=amd64 ./build.sh
```

Comment:
Hi -- are there 2 flags for darwin amd64?

Reply:
That second arch flag for darwin is `arm64` not `amd64` i.e Apple Silicon or M1 cpus:
https://tip.golang.org/doc/go1.16#ports
https://github.com/etcd-io/etcd/pull/14436

Reply:
yep etcd 3.5.5 has m1 support now